packages/gnome_keyring.rb

```diff
@@ -1,6 +1,6 @@
-require 'package'
+require 'buildsystems/autotools'
 
-class Gnome_keyring < Package
+class Gnome_keyring < Autotools
   description 'GNOME password and secret manager'
   homepage 'https://www.gnome.org'
   version '3.36.0-1'
```

Comment:
Update to [46.1](https://download.gnome.org/sources/gnome-keyring/46/gnome-keyring-46.1.tar.xz)?